tests/ami/test_basic.py

```diff
@@ -128,6 +129,18 @@ async def test_register_fields(
     assert registration.created_at < registration_date
 
 
+async def test_unregister(
+    test_client: TestClient[Litestar],
+    db_session: AsyncSession,
+    registration: Registration,
+) -> None:
+    response = test_client.delete("/api/v1/registrations/1")
```

Comment:
For extra clarity, I'd add a `assert len(all_registrations) == 1` at the beginning, what do you think?